lib/srv/db/cloud/users/users_test.go

```diff
@@ -119,12 +119,23 @@ func TestUsers(t *testing.T) {
 		// Validate db6 is same as before.
 		requireDatabaseWithManagedUsers(t, users, db6, []string{"alice", "bob"})
 	})
+
+	t.Run("new database with same name", func(t *testing.T) {
+		newDB6 := mustCreateRDSDatabase(t, "db6")
+		users.setupDatabaseAndRotatePasswords(ctx, newDB6)
+
+		// Make sure no users are cached for "db6".
+		_, err := users.GetPassword(context.TODO(), db6, "alice")
```

Comment:
```suggestion
		_, err := users.GetPassword(context.Background(), db6, "alice")
```

There's nothing "to do" here.

Same for similar occurrences.